onos-config/templates/deployment.yaml

```diff
@@ -25,6 +25,8 @@ spec:
         app: onos
         type: config
         resource: {{ template "onos-config.fullname" . }}
+        proxy.atomix.io/inject: "true"
+        proxy.atomix.io/profile: {{ template "onos-config.fullname" . }}
         {{- include "onos-config.selectorLabels" . | nindent 8 }}
       annotations:
         proxy.atomix.io/inject: "true"
```

Comment:
do these same statements still need to be in `annotations` also?

Reply:
Yeah, we're keeping them in both the labels and the annotations for now to preserve backward compatibility (so the charts will work with all versions of the Atomix controllers), but ultimately we're moving to using labels in the controllers, so eventually we can remove the annotations (once all our systems have been upgraded to the latest controller versions).